tests/informatics/test_predictor_evaluation_result.py

```diff
@@ -38,21 +38,32 @@ def test_holdout_serde(example_holdout_result, example_holdout_result_dict):
     round_trip = PredictorEvaluationResult.build(json.loads(json.dumps(example_holdout_result_dict)))
     assert example_holdout_result.evaluator == round_trip.evaluator
 
-def test_evaluator(example_cv_result, example_cv_evaluator_dict):
+
+def test_ev_evaluator(example_cv_result, example_cv_evaluator_dict):
     args = example_cv_evaluator_dict
     del args["type"]
     expected = CrossValidationEvaluator(**args)
+    assert expected.responses == set(example_cv_evaluator_dict['responses'])
     assert example_cv_result.evaluator == expected
     assert example_cv_result.evaluator != 0  # make sure eq does something for mismatched classes
 
 
+def test_holdout_set_evaluator(example_holdout_result, example_holdout_evaluator_dict):
+    args = example_holdout_evaluator_dict
+    del args["type"]
+    expected = HoldoutSetEvaluator(**args)
+    assert expected.responses == set(example_holdout_evaluator_dict['responses'])
+    assert example_holdout_result.evaluator == expected
+    assert example_holdout_result.evaluator != 0  # make sure eq does something for mismatched classes
+
+
```

Comment:
This type was never formally tested, just implicitly through the workflow reports.